L1Trigger/L1THGCal/test/HGCalTriggerGeomTesterV9Imp3.cc

```diff
@@ -1137,10 +1137,11 @@ void HGCalTriggerGeomTesterV9Imp3::fillTriggerGeometry()
     moduleZ_ = position.z();
 
     moduleSide_ = id.zside();
-    moduleSubdet_ = id.subdetId();
+    moduleSubdet_ = id.triggerSubdetId();
     moduleLayer_ = id.layer();
     module_ = 0;
-    if (id.subdetId() == ForwardSubdetector::HGCHEB || id.subdetId() == ForwardSubdetector::HFNose) {
+    if (moduleSubdet_ == HGCalTriggerSubdetector::HGCalHScTrigger ||
+        moduleSubdet_ == HGCalTriggerSubdetector::HFNoseTrigger) {
```

Comment:
I may have missed this before: `HFNose` is silicon, so it doesn't have `ieta`, `iphi`.

Reply:
Thanks, removed.